examples/README.md

```diff
@@ -31,6 +33,13 @@ integrated with a datashading pipeline. To start, run:
 python dashboard/dashboard.py --config dashboard/nyc_taxi.yml
 ```
 
+Also available, is the [2010 Census Synthetic People dataset in `castra` format](http://s3.amazonaws.com/bokeh_data/census.castra.tar.gz)
+
+This dataset can be visualized in the dashboard example using:
```

Comment:
After unpacking this file in example/data/, this dataset can be visualized in the dashboard example using:

?